js/functions.js

```diff
@@ -8604,6 +8607,19 @@ function searchJackett(){
 		} );
 
 }
+function jackettDownload(url) {
+	let blackholeLink=url.substring(url.indexOf("/bh/"));
```

Comment:
Not quite sure where to strip the link. Here or in the API. Wanted to not use the complete URL but instead still use the configured Jackett API URL. I have a feeling it might be a miss-match otherwise.